tests/conftest.py

```diff
@@ -2603,6 +2603,15 @@ def nallo_dir(tmpdir_factory, apps_dir: Path) -> str:
     return Path(nallo_dir).absolute().as_posix()
 
 
+@pytest.fixture(scope="function")
+def nallo_mock_config(nallo_dir: Path, nallo_case_id: str) -> None:
```

Comment:
This is technically not a mock
```suggestion
def nallo_config(nallo_dir: Path, nallo_case_id: str) -> None:
```

Reply:
Sounds good, I've changed it for Nallo and the other nf pipelines.